.changeset/honest-students-clean.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/integration': patch
```

Comment:
feel free to split it into multiple changesets if the description does not fit all of them since this text goes into the individual changelog for each package :)

Reply:
thanks! I will adjust it

Reply:
seems like I also missed `@backstage/plugin-scaffolder-backend`. I think I did some more changes after generating the changeset.